kola/tests/ignition/filesystem.go

```diff
@@ -253,6 +257,53 @@ func init() {
 		UserData:    swapConfigV2_1,
 		Distros:     []string{"cl"},
 	})
+
+	swapActivation := conf.ContainerLinuxConfig(fmt.Sprintf(`storage:
+  files:
+  - path: /etc/sysctl.d/80-swappiness.conf
+    filesystem: root
+    mode: 0644
+    contents:
+      inline: "vm.swappiness=10"
+
+systemd:
+  units:
+    - name: var-vm-swapfile1.swap
+      enabled: true
+      contents: |
+        [Unit]
+        Description=Turn on swap
+        Requires=create-swapfile.service
+        After=create-swapfile.service
+
+        [Swap]
+        What=%[1]s
+
+        [Install]
+        WantedBy=multi-user.target
+    - name: create-swapfile.service
+      contents: |
+        [Unit]
+        Description=Create a swapfile
```

Comment:
I would add `DefaultDependencies=no` here (not on the .swap unit as suggested in the issue)